ExtractUserDataActivity/handler.ts

```diff
@@ -189,8 +214,15 @@ export const createExtractUserDataActivityHandler = (
   notificationStatusModel: NotificationStatusModel,
   profileModel: ProfileModel,
   senderServiceModel: SenderServiceModel,
-  blobService: BlobService
-  // tslint:disable-next-line: no-big-function
+  blobService: BlobService,
+  userDataContainerName: NonEmptyString,
+  createCompressedStream: (
+    // tslint:disable-next-line: no-any
+    data: Record<string, any>,
+    password: NonEmptyString
+  ) => stream.Readable
```

Comment:
it would have been useful to pass this as a parameter in case you'd used some generic typings 
(ie. () => stream.Readable) but since this is the exact type of `createCompressedStream` then you can just import the method and use it directly without passing it as a parameter

Reply:
I did it mainly for testing, so it's easier to mock such function

Reply:
we'll find a way to mock it anyway ie `jest.mock("../zip")`